config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-2-upgrades.yaml

```diff
@@ -325,6 +325,53 @@ periodics:
             cpu: 7300m
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.2
-    testgrid-tab-name: capi-e2e-release-1-2-1-24-latest
+    testgrid-tab-name: capi-e2e-release-1-2-1-24-1-25
+    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
+    testgrid-num-failures-to-alert: "4"
+
+- name: periodic-cluster-api-e2e-workload-upgrade-1-25-latest-release-1-2
+  interval: 24h
+  decorate: true
+  decoration_config:
+    gcs_credentials_secret: "" # Use workload identity for uploading artifacts
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: release-1.2
+    path_alias: sigs.k8s.io/cluster-api
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    serviceAccountName: prowjob-default-sa
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220903-da891f21aa-1.24
```

Comment:
q: should this be
```suggestion
    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220903-da891f21aa-1.25
```

Reply:
Should stay on 1.24 as release-1.2 is using 1.24 dependencies + the go version used in v1.24 (1.18)